### PR TITLE
Transaction info

### PR DESCRIPTION
**Pull request by rohit77p**
Updated description of `transactions` and `refundTransactionDetails` array to include notes about MoR
Updated vendor list for cars
Minor updates to descriptions for language use.

### CODE REVIEW

File: apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib [hunk 1]
```diff
@@ -33,7 +33,11 @@
   + Canceled
 + cancellationDetails (cancellationDetails) - Contains information related to the cancellation request.
 + include paymentStatus
-+ refundTransactionDetails (array[refundTransactionDetails]) - Contains information about the refund transaction for the cancellation request.
++ refundTransactionDetails (array[refundTransactionDetails]) - Contains information about the refund transaction for the cancellation request.<br>
+<b>Notes:</b>
+  + The information in the `refundTransactionDetails` array is returned only if Connexions Loyalty is the Merchant of Records (MoR) and there is a refund amount to be processed.
```
Comment (mirairani) [inline apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib]: delete extra S after Record

File: apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib [hunk 2]
```diff
@@ -33,7 +33,11 @@
...
+  + The `refundTransactionDetails` array is returned as an empty array if Connexions Loyalty is the MoR, but there is no refund amount to be processed for the canceled booking (0 refund). In this case, a message is returned indicating that there is no refund for the canceled booking.
```
Comment (mirairani) [inline apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib]: add condition first then the statement.

File: apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib [hunk 3]
```diff
@@ -33,7 +33,11 @@
...
+  + The `refundTransactionDetails` array is returned as an empty array if Connexions Loyalty is not the MoR, irrespective of whether there is a refund amount to be processed for the canceled booking or not.
```
Comment (mirairani) [inline apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib]: add condition first then the statement/result.

File: apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib [hunk 4]
```diff
@@ -93,7 +97,10 @@
 + bookingDetails (bookingDetails) - Contains the booking details.
 + errors (array[errors]) - Contains the list of errors that occurred during the booking process.
 + warnings (array[warnings]) - Contains the list of warnings that were encountered during the booking process.
-+ transactions (array[transactions]) - Contains the list of payment-related transactions made for the booking, such as authorization, void, or settlement.
++ transactions (array[transactions]) - Contains the list of payment-related transactions made for the booking, such as authorization, void, or settlement.<br>
```
Comment (mirairani) [inline apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib]: such as the authorization, void, or settlement transactions.<br>

without the last word can get this line confused as something else.

File: apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib [hunk 5]
```diff
@@ -93,7 +97,10 @@
...
+<b>Notes:</b>
+  + The information in the `transactions` array is returned only if Connexions Loyalty is the Merchant of Record (MoR).
```
Comment (mirairani) [inline apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib]: only when - instance

File: apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib [hunk 6]
```diff
@@ -93,7 +97,10 @@
...
+  + The `transactions` array is returned as an empty array if Connexions Loyalty is not the MoR.
```
Comment (mirairani) [inline apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib]: add condition first then statement

File: apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib [hunk 7]
```diff
@@ -197,7 +204,7 @@
 + include hotelIdReqBook
 + rooms (array[roomsbook], required) - Contains information about the room to be booked.<br><br>**Note:** Although this object is an array, we currently support booking only *one* room at a time. If you want to book more than one room, you need to call the `Get Price` API for the next room you want to book and then the `Initiate Booking` API. For more information on booking multiple rooms, see the <a href="../overview#multiroom" target="_blank">Book Multiple Rooms in One Session</a> section.
 + paymentBreakup (array[paymentBreakup], required) - Contains the list of payment details for the booking.<br><br>
-If Connexions Loyalty is Merchant of Record, the `paymentBreakup.type` field must be set as `Card` and the `paymentBreakup.paymentMethodRefId` field is required.<br><br>
+If Connexions Loyalty is Merchant of Record, the `paymentBreakup.type` field must be set to `Card` and the `paymentBreakup.paymentMethodRefId` field is required.<br><br>
```
Comment (mirairani) [inline apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib]: is the Merchant of Record

Comment (mirairani) [inline apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib]: Card or card?

File: apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib [hunk 8]
```diff
@@ -208,7 +215,7 @@ Sample JSON with only the required fields provided if Connexions Loyalty is Merc
             "currency": "USD"
         }
     ]</pre><br>
-If Connexions Loyalty is NOT Merchant of Record, the `paymentBreakup.type` field must be set as `Cash` and the `paymentBreakup.paymentMethodRefId` field must not be provided.<br><br>
+If Connexions Loyalty is NOT Merchant of Record, the `paymentBreakup.type` field must be set to `Cash` and the `paymentBreakup.paymentMethodRefId` field must not be provided.<br><br>
```
Comment (mirairani) [inline apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib]: Cash or cash?

Comment (mirairani) [inline apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib]: is NOT the

Comment (mirairani) [inline apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib]: provided where? in the ABC object?

File: apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib [hunk 9]
```diff
@@ -219,7 +226,7 @@ Sample JSON with only the required fields provided if Connexions Loyalty is NOT
         }
     ]</pre><br>
 + paymentMethod (paymentMethod) - Contains the list of payment details for the booking.<br><br>
-<b>Note:</b> The `paymentMethod` object is required only if Connexions Loyalty is Merchant of Record (MoR), in which case the `paymentBreakup.type` field in set as `Card`. If the `paymentBreakup.type` field is set as `Cash`, the `paymentMethod` object must not be passed. 
+<b>Note:</b> The `paymentMethod` object is required only if Connexions Loyalty is Merchant of Record (MoR). Additionally, in this case, you must set the `paymentBreakup.type` field to `Card`. If the `paymentBreakup.type` field is set to `Cash`, the `paymentMethod` object must not be passed.
```
Comment (mirairani) [inline apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib]: the Merchant

Comment (mirairani) [inline apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib]: only when

Comment (mirairani) [inline apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib]: put condition first then statement

File: apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib (hunk 5 above)
Comment (mirairani) [inline apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib]: condition first

File: apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib [hunk 10]
```diff
@@ -304,7 +311,7 @@ Irrespective of whether the `/rates[]/allGuestsInfoRequired` field in the `Get P
...
-+ type (enum, required)- Indicates the type of payment that is being used to make the booking.<br><br><b>Important:</b><br> - If Connexions Loyalty is Merchant of Record, the `paymentBreakup.type` field must be set as `Card` and the `paymentBreakup.paymentMethodRefId` field is required.<br> - If Connexions Loyalty is NOT Merchant of Record, the `paymentBreakup.type` field must be set as `Cash` and the `paymentBreakup.paymentMethodRefId` field must not be provided.<br>
++ type (enum, required)- Indicates the type of payment that is being used to make the booking.<br><br><b>Important:</b><br> - If Connexions Loyalty is Merchant of Record, the `paymentBreakup.type` field must be set to `Card` and the `paymentBreakup.paymentMethodRefId` field is required.<br> - If Connexions Loyalty is NOT Merchant of Record, the `paymentBreakup.type` field must be set to `Cash` and the `paymentBreakup.paymentMethodRefId` field must not be provided.<br>
```
Comment (mirairani) [inline apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib]: <b>Important:</b><br> - If Connexions Loyalty

remove the dash

Comment (mirairani) [inline apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib]: the Merchant of Record

Comment (mirairani) [inline apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib]: the Merchant

Comment (mirairani) [inline apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib]: Cash or cash

Comment (mirairani) [inline apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib]: Card or card

File: apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib (hunk 1 above)
Comment (rohit77p) [reply apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib]: Done.

File: apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib (hunk 2 above)
Comment (rohit77p) [reply apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib]: Done.

File: apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib (hunk 4 above)
Comment (rohit77p) [reply apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib]: Done.

File: apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib (hunk 6 above)
Comment (rohit77p) [reply apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib]: Done.

File: apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib (hunk 7 above)
Comment (rohit77p) [reply apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib]: Done.

Comment (rohit77p) [reply apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib]: Card. It's an enum list. Card and Cash.

File: apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib (hunk 8 above)
Comment (rohit77p) [reply apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib]: Cash. It's an enum list. Card and Cash.

Comment (rohit77p) [reply apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib]: Done.

Comment (rohit77p) [reply apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib]: ....must not be provided in the `paymentBreakup` array. Updated.

File: apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib (hunk 9 above)
Comment (rohit77p) [reply apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib]: Done.

Comment (rohit77p) [reply apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib]: Updated.

File: apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib (hunk 5 above)
Comment (rohit77p) [reply apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib]: Done.

File: apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib (hunk 10 above)
Comment (rohit77p) [reply apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib]: Done.

Comment (rohit77p) [reply apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib]: Done.

Comment (rohit77p) [reply apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib]: Cash. Enum list.

Comment (rohit77p) [reply apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib]: Card. Enum list.

File: apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib (hunk 3 above)
Comment (rohit77p) [reply apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib]: Updated.

**Comment by rohit77p** [comment]
Changes made as suggested. Please review and approve.

**Pull request merged**